fission-core/library/Fission/Key/Asymmetric/Algorithm/Types.hs

```diff
@@ -21,8 +21,7 @@ instance Arbitrary Algorithm where
 instance ToJSON Algorithm where
   toJSON = String . \case
     RSA2048 -> "RS256" -- Per the JWT Spec (RFC 7519)
-    Ed25519 -> "Ed25519"
-    -- TODO Ed25519 -> "EdDSA"
+    Ed25519 -> "EdDSA"
```

Comment:
@walkah this is mostly us failing to read the updated JWT spec a year ago

Reply:
You'll notice that on line 30 we maintain backwards compat when reading these in, but not when creating new UCANs.